fluent-bit/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create }}
```

Comment:
This way of writing makes it possible to create only serviceaccount, but what kind of use case ?
FLY: https://github.com/helm/charts/pull/21204/files#diff-a196c8e95b4eb0a3e01b1b782632008bR1

Reply:
@cw-sakamoto 
I don't have a use case.
However, in the latest version, it looks like this when created with `helm create`, so we haven't changed it for clarity.